client/cmd/simnet-trade-tests/run

```diff
@@ -86,7 +86,7 @@ dcrdextt - Decred RPC wallet and Ethereum wallet with dextt.eth test token on ma
 
 ---------------------
 
-One or more of the following tests can be run by specifying one or more -t flags. Default is ['success'].
+One or more of the following tests can be run by specifying one or more -t flags. Default is ['success']. All tests can be run win --all.
```

Comment:
win -> with

I like the `except` switch you added.  Will you add a short sentence about that too?

Reply:
Corrected and added.